docs/src/operators/maintenance.md

```diff
@@ -14,6 +14,16 @@ We increment our Gardener dependency version by version following the Gardener u
 
     If you use the Gardener integration of metal-stack do not skip any patch releases. You may skip patch releases if you depend on metal-stack only, but we recommend to just deploy every patch release one by one for the best possible upgrade experience.
 
+## Releases
+
+Before upgrading your metal-stack installation, review the release notes carefully - they contain important information on required pre-upgrade actions and notable changes. These notes are currently shared via a dedicated Slack channel and are also available in the release on GitHub. Once you are prepared, you can deploy a new metal-stack version by updating the `metal_stack_release_version` variable in your Ansible configuration and trigger the corresponding deployment jobs in your CI.
+metal-stack uses prebuilt system images for firewalls and worker machines, which are downloaded from images.metal-stack.io. In offline or air-gapped setups, these images must be manually downloaded in advance and uploaded to your local S3-compatible storage. Ensure that the image paths and metadata are correctly maintained so the system can retrieve them during provisioning.
+If you are using metal-stack in combination with Gardener, make sure to reconcile all shoot clusters after upgrading metal-stack to ensure they remain in a consistent and fully functional state.
```

Comment:
I am not sure if this operators need to do this. Typically, shoot clusters have their own maintenance time windows and we went over to just let shoot clusters reconcile in these user-defined time windows.

The only thing I could suggest is:

```suggestion
If you are using metal-stack in combination with Gardener and you do not run pre-production stages, we advise running some basic functional tests after upgrading metal-stack to ensure the installation is in a fully functional state (e.g. reconciling a bunch of shoot clusters with evaluation purpose, creating and deleting a shoot cluster).
```